### PR TITLE
Added eslint

### PR DESCRIPTION
**Pull request by Nitive**
Added eslint and resolve eslint errors.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,6 +4,7 @@
   "description": "chat",
   "main": "index.js",
   "scripts": {
+    "lint": "./node_modules/.bin/eslint --ext .js --ext .jsx app/; exit 0",
```
Comment (SeqviriouM) [inline package.json]: You check lint only in app folder?

Comment (Nitive) [inline package.json]: Ok, I will add folder `server/`

Comment (Nitive) [inline package.json]: done

Comment (SeqviriouM) [inline package.json]: Ok, accepted

**Comment by dimaip** [comment]
+1 from me. Great stuff.
We could also run all tests from pre-commit hooks.

**Comment by SeqviriouM** [comment]
+1

**Comment by dimaip** [comment]
Merge it asap, or there will be a lot of conflicts

**Comment by Faraser** [comment]
+1

**Comment by hotspurs** [comment]
+1

**Pull request merged**